src/index.js

```diff
@@ -55,7 +66,7 @@ async function handleRequest(event) {
   });
 
   const optimizelyUserContext = optimizelyClient.createUserContext(
-    "USER_ID_HERE",
+    userId,
```

Comment:
Although we shouldn't run into the issue with the current logic, to help future-proof the code should we validate and/or enforce `userId` to be a string?